docs/Board - PyroDroneF4.md

```diff
@@ -0,0 +1,47 @@
+# Board - PYRODRONEF4
+
```

Comment:
Thanks for adding this. Can you please move it into the `docs/boards/` directory? (You'll have to update to latest in `master`, the directory is new.)